### ISSUE TITLE
page not loading

### ISSUE THREAD

**Issue by prav843** [theankurkedia/newsbuzz#4]
cannot fetch news at all
stuck =>loading screen
![screenshot-1522348215000](https://user-images.githubusercontent.com/37571579/38106709-96691e06-33af-11e8-9b51-9d47fe348f49.jpg)

**Comment by davigun** [comment]
Yes, the same as me, it's throwing an exception about the view is being null. hmmm any idea why?

**Comment by erdie** [comment]
Same with me

**Comment by investcode** [comment]
Same with me

**Comment by investcode** [comment]
http://joxi.ru/xAeOkQlSpXXoPm

**Comment by davigun** [comment]
Whats that ??

**Comment by investcode** [comment]
page not loading https://user-images.githubusercontent.com/37571579/38106709-96691e06-33af-11e8-9b51-9d47fe348f49.jpg

**Comment by shankarvannala** [comment]
cannot fetch news at all , stucking at loading screen
